scout/dashboard.md

```diff
@@ -66,19 +66,41 @@ Click the **View recommended fixes** button to see instructions to apply the rec
 
 ## Packages and dependencies
 
-![A screenshot of the Docker Scout packages and dependencies list](./images/dashboard-pandd.png)
+The **Packages and dependencies** tab shows all **Packages** and **Base images** across images in an organization.
```

Comment:
```suggestion
The **Packages and dependencies** tab shows all packages and base images across images in an organization.
```

Reply:
I think it was correct the first time around

Reply:
@aevesdocker Yeah, I did this so I could refer to the tabs without mentioning them twice? 🤷 . Waddy think?